lib/src/models/item_model.dart

```diff
@@ -0,0 +1,21 @@
+import 'result_model.dart';
+
+class ItemModel {
+  int _page;
+  List<Result> _results = [];
+
+  ItemModel.fromJson(Map<String, dynamic> parsedJson) {
+    _page = parsedJson['page'];
```

Comment:
what happens if for any means this is null ? it will break your code, remember to always cover yourself from these cases